src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -1423,22 +1409,20 @@ private static string FormatDictionary(IDictionary content)
 
         private ErrorRecord GetValidationError(string msg, string errorId)
         {
-            var ex = new ValidationMetadataException(msg);
-            var error = new ErrorRecord(ex, errorId, ErrorCategory.InvalidArgument, this);
-            return error;
+            ValidationMetadataException ex = new(msg);
+            return new ErrorRecord(ex, errorId, ErrorCategory.InvalidArgument, this);
         }
 
         private ErrorRecord GetValidationError(string msg, string errorId, params object[] args)
         {
             msg = string.Format(CultureInfo.InvariantCulture, msg, args);
-            var ex = new ValidationMetadataException(msg);
-            var error = new ErrorRecord(ex, errorId, ErrorCategory.InvalidArgument, this);
-            return error;
+            ValidationMetadataException ex = new(msg);
+            return new ErrorRecord(ex, errorId, ErrorCategory.InvalidArgument, this);
         }
 
         private string GetBasicAuthorizationHeader()
```

Comment:
I noticed we use `GetBasicAuthorizationHeader()` but Basic Authentication is supported by the Credential parameter it seems redundant, I think we could remove `GetBasicAuthorizationHeader()` with no breaking changes
test:
```ps1
$password = ConvertTo-SecureString 'testpassword' -AsPlainText -Force
$credential = New-Object System.Management.Automation.PSCredential ('testuser', $password)
Invoke-WebRequest "https://httpbin.org/basic-auth/testuser/testpassword" -Credential $credential
```

Reply:
Perhaps HttpClientHandler uses Basic auth as default but I don't see this in docs. I don't see the point in changing something unless something is working wrong.